dir.targets

```diff
@@ -1,5 +1,22 @@
 <Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  
+
+  <UsingTask TaskName="FixFilePath" TaskFactory="CodeTaskFactory" AssemblyFile="$(MSBuildToolsPath)\Microsoft.Build.Tasks.v12.0.dll">
```

Comment:
Per the conversation, we really should avoid adding inline tasks if at all possible.